src/Container.js

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+
+const maxWidth = props => props.maxWidth ? { maxWidth: `${props.maxWidth}px` } : { maxWidth: props.theme.maxContainerWidth }
```

Comment:
Would something like this belong in `styled-system` @jxnblk ?

Reply:
This could be handled with the generic [`style`](https://github.com/jxnblk/styled-system#style) utility, but I don't mind this here as it's fairly readable as-is